src/js/os.js

```diff
@@ -16,6 +16,41 @@ exports.freemem = function() {
   return native.getFreeMem();
 };
 
+exports.getPriority = function(pid) {
+  if (pid === undefined)
+    pid = 0;
+  else
+    pid = Number(pid);
+
+  var err = native.getPriority(pid);
+  if (err instanceof Error) {
+    throw err;
+  }
+  return err;
+};
+
+exports.setPriority = function(pid, priority) {
+  if (priority === undefined) {
+    priority = pid;
+    pid = 0;
+  } else {
+    pid = Number(pid);
+  }
+
+  if (typeof priority !== 'number') {
+    throw new TypeError('priority must be a number');
+  }
+  if (priority > 19) {
```

Comment:
use `UV_PRIORITY_LOW` and `UV_PRIORITY_HIGHST` defined  from  UV instead of literal ?

Reply:
Actually those defines are for using at Windows, I still leave those on uv.h is to acquire consistence.

Reply:
seems those defines are using at Linux too, see setPriority [doc](https://linux.die.net/man/2/setpriority)

Reply:
Could you please reference the code/lines in details? Can not search any that defines from your link :(